.lgtm.yml

```diff
@@ -8,6 +8,8 @@ extraction:
         - cmake
         - libssl-dev
     after_prepare:    # Customizable step used by all languages.
+      # installs cmake!
+      - cmake --version
```

Comment:
version twice?

Reply:
check before and after. maybe at some point we don't need the manual install anymore ;)